packages/amplication-server/src/core/assistant/assistant.service.ts

```diff
@@ -665,5 +674,58 @@ export class AssistantService {
             : "Entity",
       }));
     },
+    getPlugins: async (args: undefined, context: AssistantContext) => {
+      const plugins = (await this.pluginCatalogService.getPlugins()).map(
+        (plugin) => omit(plugin, ["__typename"])
+      );
+      this.logger.debug(
+        `Chat: Plugins: ${JSON.stringify(plugins)}`,
+        null,
+        context
+      );
+      return plugins;
+    },
+    installPlugins: async (
```

Comment:
we are missing the unit tests here. Try the edge case of `this.pluginCatalogService.getPluginWithLatestVersion` returning  `undefined`